bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/MeasureCapability.java

```diff
@@ -42,28 +43,20 @@
  *
  */
 @NonNullByDefault
-public class MeasureCapability extends RestCapability<WeatherApi> {
+public class MeasureCapability extends BufferedWeatherCapability {
+    private final static int MIN_DATA_VALIDITY_MIN = 30;
```

Comment:
Isn't 30 minutes too much ?

Reply:
For measures the minimum timescale is 30 minutes. [These results are statistic analysis of past data, so I thought it would be far enough.](https://dev.netatmo.com/apidocumentation/weather#getmeasure)